apps/emqx_mix_utils/lib/mix/tasks/emqx.ct.ex

```diff
@@ -99,6 +101,13 @@ defmodule Mix.Tasks.Emqx.Ct do
     end
   end
 
+  def sum_results(l) do
+    Enum.reduce(l, fn {success, failed, {skipped, auto_skipped}},
```

Comment:
to avoid weird crashes in case no suites/cases are enumerated

```suggestion
    init_acc = {0, 0, {0, 0}}
    Enum.reduce(l, init_acc, fn {success, failed, {skipped, auto_skipped}},
```

Reply:
Thanks, fixed.